test/test_helper.exs

```diff
@@ -1 +1,2 @@
-ExUnit.start()
+Application.ensure_all_started(:nerves_ssh)
```

Comment:
You got this working! I think that https://github.com/nerves-project/nerves_ssh/blob/main/mix.exs#L27-L31 can be deleted then, right?

Reply:
I totally forgot about this and was wondering why the application wasn't being started by default 🤦 
Fixed